runtime/src/main/java/uk/ac/manchester/tornado/runtime/tasks/TornadoTaskSchedule.java

```diff
@@ -104,65 +54,80 @@
 import uk.ac.manchester.tornado.runtime.sketcher.TornadoSketcher;
 import uk.ac.manchester.tornado.runtime.tasks.meta.ScheduleMetaData;
 
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.lang.reflect.Array;
+import java.lang.reflect.Method;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Consumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static uk.ac.manchester.tornado.runtime.TornadoCoreRuntime.getTornadoRuntime;
+import static uk.ac.manchester.tornado.runtime.common.RuntimeUtilities.humanReadableByteCount;
+import static uk.ac.manchester.tornado.runtime.common.RuntimeUtilities.isBoxedPrimitiveClass;
+import static uk.ac.manchester.tornado.runtime.common.Tornado.VM_USE_DEPS;
+import static uk.ac.manchester.tornado.runtime.common.Tornado.warn;
+
 /**
  * Implementation of the Tornado API for running on heterogeneous devices.
  */
 public class TornadoTaskSchedule implements AbstractTaskGraph {
 
+    /**
+     * Options for Dynamic Reconfiguration
+     */
+    private static final boolean EXEPERIMENTAL_MULTI_HOST_HEAP = false;
+    private static final int DEFAULT_DRIVER_INDEX = 0;
+    private static final int PERFORMANCE_WARMUP = 3;
+    private final static boolean TIME_IN_NANOSECONDS = Tornado.TIME_IN_NANOSECONDS;
+    private static final String TASK_SCHEDULE_PREFIX = "XXX";
+    private static final ConcurrentHashMap<Policy, ConcurrentHashMap<String, HistoryTable>> executionHistoryPolicy = new ConcurrentHashMap<>();
+    private static final int HISTORY_POINTS_PREDICTION = 5;
+    private static final boolean USE_GLOBAL_TASK_CACHE = false;
+    private static ConcurrentHashMap<Integer, TaskSchedule> globalTaskScheduleIndex = new ConcurrentHashMap<>();
+    private static int baseGlobalIndex = 0;
+    private static AtomicInteger offsetGlobalIndex = new AtomicInteger(0);
+    private final CompileInfo COMPILE_ONLY = new CompileInfo(true, false);
```

Comment:
These fields are not related to dynamic reconfiguration. Why is this reordering?